### DISCUSSION TITLE
Custom logo not visible after introducing translations

### DISCUSSION THREAD

**Discussion by nikolabul** [open-sdg/open-sdg#1743] [Q&A] [answered]
What's the right way to organize assets/img/ structure for the custom SDG_logo to be visible in different languages?

My current structure for two languages is:
assets/img/logo/cnr/SDG_logo.png
assets/img/logo/en/SDG_logo.png

The custom logo is ignored and the default one is displayed in the frontpage.

**Comment by brockfanning** [comment, marked as answer]
@nikolabul I believe that would need to be handled in the [logos site configuration](https://open-sdg.readthedocs.io/en/latest/configuration/#logos). You can use any folder/file structure you want, but it needs to be configuration in the "logos" setting.
